### ISSUE TITLE
Failed to load the linter-coffee-variables package

### ISSUE THREAD

**Issue by nickdima** [markwainwright/atom-linter-coffee-variables#1]
1. install the package
2. close and reopen Atom 

**Atom Version**: 1.1.0
**System**: Mac OS X 10.10.5
**Thrown From**: [linter-coffee-variables](https://github.com/markwainwright/atom-linter-coffee-variables) package, v0.4.0
### Stack Trace

Failed to load the linter-coffee-variables package

```
At Cannot read property 'debug' of undefined

TypeError: Cannot read property 'debug' of undefined
    at Object.<anonymous> (/Users/nick/.atom/packages/linter-coffee-variables/lib/debug.coffee:2:50)
    at Object.<anonymous> (/Users/nick/.atom/packages/linter-coffee-variables/lib/debug.coffee:1:1)
    at Module._compile (module.js:428:26)
    at Object.defineProperty.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:169:21)
    at Module.load (module.js:353:32)
    at Function.Module._load (module.js:308:12)
    at Module.require (module.js:363:17)
    at require (module.js:382:17)
    at Object.<anonymous> (/Users/nick/.atom/packages/linter-coffee-variables/lib/linter-coffee-variables.coffee:1:9)
    at Object.<anonymous> (/Users/nick/.atom/packages/linter-coffee-variables/lib/linter-coffee-variables.coffee:1:1)
```
### Config

``` json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "disabledPackages": [
      "terminal-status",
      "change-case",
      "debugger",
      "web-browser",
      "js2coffee",
      "filetype-color",
      "atom-color-highlight"
    ],
    "themes": [
      "one-dark-ui",
      "oceanic-next"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-handlebars, v1.2.0
atom-pair, v2.0.10
devdocs, v0.2.1
file-icons, v1.6.11
highlight-selected, v0.10.1
language-babel, v0.15.12
language-cjsx, v0.3.0
language-elm, v1.2.0
language-swift, v0.4.0
linter, v1.10.0
linter-coffee-variables, v0.4.0
linter-coffeelint, v1.1.2
linter-eslint, v3.2.1
oceanic-next, v0.1.2
react-snippets, v0.2.0
regex-railroad-diagram, v0.10.4
remote-atom, v1.2.7
set-syntax, v0.3.0

# Dev
No dev packages
```

**Comment by nickdima** [comment]
I think the problem is that at startup the config might not be available. Check this thread: https://discuss.atom.io/t/atom-config-get-does-not-return-default-value/15982/5

**Comment by markwainwright** [comment]
@nickdima:
Ah yeah, that makes sense. Thanks for pointing me in the right direction there.
I'll fix this ASAP – or feel free to submit a PR.

**Comment by markwainwright** [comment]
I can't reproduce this, but I've just released v0.4.1 with an attempted fix.
Please update and re-open this issue if you're still having problems! Thanks.

**Comment by nickdima** [comment]
Fixed, thanks! This is quite useful.
